.gitignore

```diff
@@ -1,4 +1,353 @@
-.idea
```

Comment:
Could you leave this line? I use Jetbrains Rider 😅

Reply:
oh ok lol, it's auto generated by vs